app/jobs/drafts/parse_import_job.rb

```diff
@@ -97,13 +99,23 @@ def load_import_csv(import, csv_path, author:)
           "posp_id": row['posp_id'],
           "posp_version": row['posp_version'],
           "message_type": row['message_type'],
+          "sktalk_class": row['sktalk_class'] || DEFAULT_SKTALK_CLASS,
           "correlation_id": uuid,
           "sender_business_reference": row['sender_business_reference'],
           "recipient_business_reference": row['recipient_business_reference'],
           "import_subfolder": row['subfolder'],
           "status": "being_loaded"
         }
       )
+
+      tags = JSON.parse(row["tags"]) if row["tags"]
+      tags&.each do |tag_name|
+        message_draft.add_cascading_tag(
+          import.box.tenant.tags.find_or_create_by!(name: tag_name) do |tag|
```

Comment:
Na urovni tenanta vies urobit asociaciu s typom SimpleTag a potom nemusis tu ten type nastavovat. 

```suggestion
          import.box.tenant.simple_tags.find_or_create_by!(name: tag_name)
```

Reply:
Nad tymto som uvazovala, ale potom to obmedzime iba na `SimpleTag`y. Viem si predstavit situaciu, ze by som tam chcela pri importe pridat napr. uz existujuci `SignedByTag` "Podpisane Jano". @jsuchal

Reply:
To si nemyslim, ze by sme mali zatial dovolit. Viem si predstavit, ze tam budu vsak, ze to ma podpisat niekto iny. Cize beriem a nechajme takto zatial.